### DISCUSSION TITLE
Source->File (Globbing)

### DISCUSSION THREAD

**Discussion by bmallya** [vectordotdev/vector#15864] [General]
Hello,
Is there a way to include multiple formats of file to be included in the search when this "source" is used?
For ex: In the /tmp directory, I want to monitor files 
1. That end with .sh.log
2. that end with _error.log

**Comment by jszwedko** [comment]
Hi @bmallya !

Yes you can provide multiple patterns like `paths = ["*.sh.log", "*_.error.log"]`. Hope this helps!

**Reply by bmallya** [reply]
Thanks for the quick response @jszwedko. This definitely helps.
